pkg/logs/tailers/windowsevent/tailer_windows.go

```diff
@@ -0,0 +1,246 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Is this the legacy tailer that was removed https://github.com/DataDog/datadog-agent/pull/19761/files#diff-94dfbe326f30fe7158d3dea7b5538f5fe82b4ecfee6ec057c2a48a0b0b9658de?

It looks like it may have beena added back by mistake during the merge.